src/panels/lovelace/views/hui-view.ts

```diff
@@ -176,11 +160,45 @@ export class HUIView extends LitElement {
             max-width: 600px;
           }
         }
+
+        .grid {
+          position: relative;
+          width: 100%;
+        }
+
+        .item {
+          display: block;
+          width: 100%;
+          margin: 5px;
+          z-index: 1;
+        }
+
+        .item.edit {
+          position: absolute;
+        }
+
+        .item.muuri-item-dragging {
+          z-index: 3;
+        }
+
+        .item.muuri-item-releasing {
+          z-index: 2;
+        }
+
+        .item.muuri-item-hidden {
+          z-index: 0;
+        }
+
+        .item-content {
+          position: relative;
+          width: 100%;
+          height: 100%;
+        }
       </style>
     `;
   }
 
-  protected updated(changedProperties: PropertyValues): void {
+  protected async updated(changedProperties: PropertyValues): Promise<void> {
```

Comment:
updated can't be `async`, make it `this.updateComplete.then(() => { /* do stuff */ });`

Reply:
Gotcha. I figured this wasn't ideal. Thank you for the alt method